index.html

```diff
@@ -699,6 +699,13 @@ <h1>Terminology</h1>
       </dt>
       <dd>An entity that can create WoT Thing Descriptions
         for a specific Thing.</dd>
+      <dt>
+        <dfn>Profile</dfn>
+      </dt>
+      <dd>A technical specification which provides a set of assertions such that any <a>Consumer</a> 
```

Comment:
Two comments here:
- do we need another definition what "out-of-the-box" mean?
- I think, the definition can be more clear that the Profile is a subset of the TD specification by providing additional assertions. 
- Proposal: A technical specification that is a subset of the Thing Description specification which provides additional set of assertions such that any <a>Consumer</a> which conforms with the those assertions is out-of-the-box interoperable with any <a>Thing</a> which also conforms with those assertions.

Reply:
@sebastiankb What does it mean to be a subset of a specification?

A profile is a collection of assertions which are neither a subset nor a superset of the assertions in the Thing Description specification, they are a separate collection of assertions with may define additional constraints and assumptions for conformant Consumers and Things.

I suspect what you're trying to say is that a Thing Description conforming to a profile can only use a subset of the features defined in the Thing Description specification, but that isn't really an accurate description either. A profile may define a subset of TD features (such as security schemes, protocol bindings, and semantic contexts) which a conformant Consumer must support, but it may also define additional assumptions (such as protocol binding defaults), some of which can't even be described declaratively in a Thing Description. That could be described as a superset of the features described in the TD specification.

This current definition of a profile has already been reviewed and landed in the WoT Profile specification, so whatever lands here will need to replace or be consistent with that definition.

Reply:
Note that I've started a discussion about a ["Profile Mechanism 2.0"](https://github.com/w3c/wot-profile/issues/285) which could be much more rigidly defined, but that is a different approach to profiles than is taken in the current specification.